---
layout: single
classes: wide
title: Research
author_profile: true
permalink: /research/genomics/
header:
  overlay_image: /assets/images/denali.jpeg
---

# GENOMICS FOR ADDRESSING BIOMEDICAL CHALLENGES
---
![](/assets/images/Research_Pic.jpg/){:width="300px"
style="float: left;margin-right: 12px;margin-top: 0px;"}

Nucleic acids, such as DNA and RNA, encode the blueprint for all life on earth. With the advent of Next-Generation sequencing (NGS) technologies, which are capable of capturing greater sequencing depths more cheaply than prior platforms, researchers have been able to sequence the genomes from the smallest life forms on earth to the entirty of the human genome. This DNA sequence data has allowed us to reveal the mechanism behind certain cancers allowing scientist to generate targeted therapies, trace the microbial source of an infectious human disease(e.g. antibiotic resistant nosocomial threats like methicillin resistant *Staphylococcus aureus* (MRSA), bioterroism agents like *Bacillus anthracis*, and food-borne outbreaks such as *Listeria monocytogenes*), and more recently identify and track the spread of variants of concern from the SARS-CoV-2 virus.

The combined high throughput and reduced cost of NGS platforms has enabled major advances in the epidemiological capabilities of sequence data for surveillance and outbreak prevention.  Despite how critical sequence data can be, it is oftentimes hard to collect and analyze real-time sequence data because of the lack of personell equipt with the skills to prepare a DNA libraries for sequencing or because of the difficulty of bringing sequencing devices, such as Illumina Sequencers, into the field. 

![](/assets/images/minion-cutout.png/){:width="300px"
style="float: left;margin-right: 12px;margin-top: 0px;"}

Oxford Nanopore technology (ONT) had produced several long-read sequencing devices with massive field-forward potential. For example, to prepare DNA libraries there is a library-prep device known as a VolTrax that comes with instructions on how to load all reagents and DNA which then carries out the entire library prep process for the user. DNA libraries can then be easily loaded onto the handheld sequencing device known as the MinION, that has the ability to basecall reads in real-time while being mapped back to a reference sequence. These user-friendly, portable, and cost effective devices have the potential to allow widespread sequencing of potential microbial threats in areas where sequencing was previously inaccessible, such as in remote regions of Alaska.

In my research we utilize and validate ONT devices in order to test the real-world potential of these user-friendly sequencing technologies for addressing public health challenges. We have tested and validated several library prep protocols, such as the use of VolTrax on environmental DNA, bacterial isolate DNA, transcriptomes, and reverse transcribed viral DNA from SARS-CoV-2 virus in order to test the potential field-forward use of these devices.  Moreover, I have used ONT technology in my research to generate completely circularized and high quality bacterial genomes with plasmids, monitor environmental samples for antibiotic resistance determinants carried on mobile genetic elements, generating the genomes of unculturable microbes, and to validate long-read methods for capturing the whole genomes of SARS-CoV-2.

---
Related Publications:

1. **Haan, T.**, Seitz, T. J., Francisco, A., Glinter, K., Gloger, A., Kardash, A., â€¦ & Drown, D. M. (2020). Complete Genome Sequences of Seven Strains of Pseudomonas spp. Isolated from Boreal Forest Soil in Interior Alaska. Microbiology resource announcements, 9(25). DOI:10.1128/MRA.00511-20

2. **Haan, T.**, McDougall, S., & Drown, D. M. (2019). Complete genome sequence of bacillus mycoides TH26, isolated from a permafrost thaw gradient. Microbiology resource announcements, 8(24). DOI:10.1128/MRA.e00507-19

3. Humphrey, J., Seitz, T., **Haan, T.**, Ducluzeau, A. L., & Drown, D. M. (2019). Complete genome sequence of Pantoea agglomerans TH81, isolated from a permafrost thaw gradient. Microbiology resource announcements, 8(1). DOI:10.1128/MRA.01486-18
